endpoints/README.md

```diff
@@ -13,6 +13,10 @@ The FHIR Endpoint Querier reads the following environment variables:
 
   Default value: endpointQuerierLog.json
 
+* **LANTERN_ENDPTQRY_QUERY_INTERVAL**: Number of minutes to wait between queries (Note: actual time between queries will be greater since the time it takes to run the queries is non-zero)
```

Comment:
state that this is in minutes.

Reply:
"Number of minutes to wait...." doesn't satisfy that?

Reply:
it definitely does... must have read over that somehow.